conda_build/skeletons/pypi.py

```diff
@@ -74,11 +75,11 @@
 # etc. is determined by EXPECTED_SECTION_ORDER.
 PYPI_META_STATIC = {
     'package': OrderedDict([
-        ('name', '{{ name|lower }}'),
+        ('name', '{{ namel }}'),
         ('version', '{{ version }}'),
     ]),
     'source': OrderedDict([
-        ('url', '/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz'),  # NOQA
+        ('url', '/packages/source/{{ namel[0] }}/{{ namel }}/{{ namel }}-{{ version }}.tar.gz'),  # NOQA
```

Comment:
The PyPI is case sensitive to the package name.

For example the following will give a 404:
https://pypi.io/packages/source/f/flask/flask-1.1.1.tar.gz
The correct URL is:
https://pypi.io/packages/source/F/Flask/Flask-1.1.1.tar.gz